packages/components/src/components/select/select.ts

```diff
@@ -47,109 +48,103 @@ import type SdPopup from '../popup/popup';
+  };
+
+  private readonly formControlController = new FormControlController(this, {
+    assumeInteractionOn: ['sd-blur', 'sd-input']
+  });
   private readonly hasSlotController = new HasSlotController(this, 'help-text', 'label');
   private readonly localize = new LocalizeController(this);
   private typeToSelectString = '';
   private typeToSelectTimeout: number;
 
-  @query('.select') popup: SdPopup;
-  @query('.select__combobox') combobox: HTMLSlotElement;
-  @query('.select__display-input') displayInput: HTMLInputElement;
-  @query('.select__value-input') valueInput: HTMLInputElement;
-  @query('.select__listbox') listbox: HTMLSlotElement;
+  @queryAssignedElements({ selector: 'sd-option' }) _optionsInDefaultSlot!: SdOption[];
+
+  @query('sd-popup') popup: SdPopup;
+  @query('[part="combobox"]') combobox: HTMLSlotElement;
+  @query('[part="display-input"]') displayInput: HTMLInputElement;
+  @query('.value-input') valueInput: HTMLInputElement;
+  @query('[part="listbox"]') listbox: HTMLSlotElement;
 
   @state() private hasFocus = false;
-  @state() displayLabel = '';
-  @state() currentOption: SdOption;
-  @state() selectedOptions: SdOption[] = [];
+  @state() hasHover = false; // we need this because Safari doesn't honor :hover styles while dragging
+  /** When `multiple` is `true` and `useTags` is `false`, the displayLabel sets the text shown in the display input. We use the localized string "Options Selected (#)" by default. */
+  @state() private displayLabel = '';
+  @state() private currentOption: SdOption;
+  @state() private selectedOptions: SdOption[] = [];
 
-  /** The name of the select, submitted as a name/value pair with form data. */
-  @property() name = '';
+  /** The default value of the form control. Primarily used for resetting the form control. */
+  @defaultValue() defaultValue: string | string[] = '';
 
   /**
-   * The current value of the select, submitted as a name/value pair with form data. When `multiple` is enabled, the
-   * value will be a space-delimited list of values based on the options selected.
+   * Indicates whether or not the select is open. You can toggle this attribute to show and hide the menu, or you can
+   * use the `show()` and `hide()` methods and this attribute will reflect the select's open state.
    */
-  @property({
-    converter: {
-      fromAttribute: (value: string) => value.split(' '),
-      toAttribute: (value: string[]) => value.join(' ')
-    }
-  })
-  value: string | string[] = '';
-
-  /** The default value of the form control. Primarily used for resetting the form control. */
-  @defaultValue() defaultValue: string | string[] = '';
+  @property({ type: Boolean, reflect: true }) open = false;
 
   /** The select's size. */
-  @property() size: 'small' | 'medium' | 'large' = 'medium';
+  @property({ reflect: true }) size: 'lg' | 'md' | 'sm' = 'lg';
+
+  /** The select's label. If you need to display HTML, use the `label` slot instead. */
+  @property() label = '';
```

Comment:
We should think about a default value, cause label is (as I know) required for accessiblity 🤔

Reply:
sure, does "Label" work?